net.sourceforge.m64py.M64Py.yml

```diff
@@ -0,0 +1,140 @@
+app-id: net.sourceforge.m64py.M64Py
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.7'
+
+command: m64py
+
+finish-args:
+  - --share=ipc
+  - --device=all
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --socket=wayland
+
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtBluetooth*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtDesigner*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtHelp*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtMultimedia*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtNetwork*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtNfc*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtOpenGL*.so
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtPositioning*.so
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtPrintSupport*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtPrintSupport*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtQml*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtQuick*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtRemoteObjects*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtSensors*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtSerialPort*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtSql*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtSvg*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtTest*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtWeb*
+  - rm -rf /app/lib/python/site-packages/PyQt6/QtXml*
+  - rm -rf /app/lib/python/site-packages/sdl2/examples
+  - rm -rf /app/lib/python/site-packages/sdl2/test
+
+modules:
+
+  - name: mupen64plus-core
+    buildsystem: simple
+    build-commands:
+      - cd projects/unix && sed -i '321,324d;315d' Makefile && make install PREFIX=/app SHAREDIR=/app/share/mupen64plus VULKAN=0 OSD=0 NEW_DYNAREC=1
```

Comment:
use `subdir: projects/unix`, `no-autogen: true` and set the make args in `make-args`

Reply:
Changed. The `sed` used here is needed because the PyQt base cleaning script will remove minizip. So pkg-config will find it and use the system library instead of the internal minizip. It doesn't help even if I include minizip in the manifest. And if I don't use the PyQt cleaning script I have to do manually a huge cleanup.